.github/workflows/run_alembic_migrations.yml

```diff
@@ -0,0 +1,64 @@
+name: Automated Migrations
+
```

Comment:
Add a comment here mention the 3 types of migrations provided:

- API
- DB
- ssh